### PR TITLE
Feature/filacms 23

### PR DESCRIPTION
**Pull request by jeremy-portable**

### CODE REVIEW

File: tests/Filament/PageResourceTest.php [hunk 1]
```diff
@@ -191,6 +191,64 @@ public function test_can_create_page_with_taxonomies(): void
...
+                'contents' => $this->createContent(),
+            ])
+            ->call('create')
+            ->assertHasNoErrors();
+
+        $page = TargetModel::where('title', 'Test Slug Title')->orderBy('id', 'desc')->first();
+
+        $this->assertEquals($page->slug, 'test-slug-title-1');
+    }
+
```
Comment (kyoungportable) [inline tests/Filament/PageResourceTest.php:251]: Please add a test for the user attempting to create a duplicate slug :)

**Review by kyoungportable — CHANGES_REQUESTED** [review]

**Review by kyoungportable — APPROVED** [review]
